connectors/src/connectors/zendesk/temporal/workflows.ts

```diff
@@ -130,8 +130,8 @@ export async function zendeskSyncWorkflow({
 
   const currentSyncDateMs = new Date().getTime();
 
-  // If we got no signal, then we're on the scheduled execution
```

Comment:
why get rid of this comment though?

Reply:
that said, it's also no longer true right ? we're on the scheduled execution if `isInitialSync` isn't true

Reply:
in fact it never really was true since people can end up here from the intitial execution by selecting nothing

Reply:
actually true, my bad